playbooks/roles/swapfile/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: check if swap file exists
+  stat:
+    path: "{{ swapfile_path }}"
+  register: swapfile_check
+- name: create swap file {{ swapfile_path }}
+  command: fallocate -l {{ swapfile_size }} {{ swapfile_path }}
+  when: not swapfile_check.stat.exists
+- name: set permissions on swap file
+  file:
+    path: "{{ swapfile_path }}"
+    mode: 0600
+- name: mkswap {{ swapfile_path }}
```

Comment:
Does this work? I've seen errors, no entry or just the literal words in the past when using this in a name.